parsers/lua.c

```diff
@@ -60,7 +60,7 @@ static bool is_a_code_line (const unsigned char *line)
 
 static bool isLuaIdentifier (char c)
 {
-	return (bool) !(isspace(c)  || c == '(' || c == ')' || c == '=');
+	return (bool) !(isspace(c)  || c == '(' || c == ')' || c == '=' || c == '.' || c == ':');
```

Comment:
Do we need this?

isLuaIdentifier() is used not only in extract_next_token() but also in extract_prev_toke().
I wonder whether the change for isLuaIdentifier() has an impact on extract_prev_toke() or not.

If you are not sure, keep isLuaIdentifier() as is.
If you are sure, could you write your conviction to the commit log.

Reply:
It is needed, since `extract_prev_token()` moves backward as long as `isLuaIdentifier()` returns true. So we add the cases for `'.'` and `':'` to have it stop as soon as it reaches one of those characters.

Reply:
I see. Thank you for your explanation.

Reply:
Updated the commit message.